research/build-reverse-dependencies/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Build reverse dependencies in Packit
+authors: mmassari
+---
+
+Already existing solutions:
+
+- [Koschei service](https://github.com/fedora-infra/koschei)
+- [Mass Prebuild CLI](https://gitlab.com/fedora/packager-tools/mass-prebuild)
+- [mini mass rebuild](https://github.com/hroncok/mini-mass-rebuild)
+- [copr-autorebuilder](https://pagure.io/copr-autorebuilder/tree/master)
+- [copr-rebuild-tools](https://github.com/fedora-copr/copr-rebuild-tools/tree/main)
```

Comment:
I assume these would probably not be a good fit since they are not included, right?

Reply:
Yes, they are *mass rebuild tools* not focused on *reverse dependencies*.
We can give them a further look if we decide to implement our own solution and we want to investigate more on how to submit a large number of builds.

Reply:
makes sense, thanks!